### ISSUE TITLE
Incompatible Datapack(s)

### ISSUE THREAD

**Issue by Coraline77447** [dainxt/WeaponThrow#5]
Hi. It's me yet again, I'm here to report yet another issue involving datapacks with this mod and why I'm reporting this about it's datapack(s) is cause I'm sick and tired of losing my worlds due to incomplete or missing data within the datapacks or outdated datapacks of some of the mods that I'm playing with which at some point I lose the world that I would be playing on and how I know this is cause when making a new world you can apply datapacks except for datapacks that mods adds, when I look in there I see some red ones that when I hover my mouse over them they say "Incompatible, made for an older version of Minecraft." When I would try to load into a modded world I get a message that says that the selected datapacks have errors and asks me to either enter safe mode (which I can't enter safe mode) or go to title screen and in the message it says that I can manually fix them which Idk how to fix incomplete, missing, outdated, or even broken datapacks to mods that has these errors.... 

P.S. I know that you've just recently released an update that fixes one problem but this one might have been slipping through the cracks for some time without us knowing about it till I'd discovered it when I was in the process of making a new world after my last one was un-joinable due to some selected datapacks that have errors that prevented me from rejoining the world again so I had to delete the world... I apologize for bringing this up after you've just fixed an issue in the mod 15+ hours ago but I'm just tryin to help ya with making this mod better by providing information that you need to make it better.... :((

**Comment by dainxt** [comment]
I'm sorry bro, this mod have no relations with that (This mod doesn't even use datapacks), as being incompatible isn't a problem, just that some mods authors forgets to change the meta-data of the buit-in datapacks. The problem resides on the datapack itself as the 1.16 minecraft version datapacks can have experimental features that can corrupt worlds. The only way is to avoid using this datapacks and probably there will be no fixes in near feature.

**Comment by Coraline77447** [comment]
I have thought that Forge had/has some experimental settings regarding world gen but I am apparently wrong to some degree and I've realized recently that I should've known that datapacks for minecraft java edition are still in its infincy or still wip (work in progress) state and I've recently downloaded and installed a mod that adds 5 new dungeons, the mod is called Dungeons Plus which is probably using datapacks which we all know that datapacks right now are still in experimental state. And I probably should start avoiding mods that adds worldgen content like ores, structures, and dimensions like this one mod called Allthemedium which is a worldgen mod that adds 3 new ores and a custom dimension. Also I don't know why that the damn game is saying that this mod has a incompatible datapack but it's apparently saying that it does , so how can I disable minecraft's datapack system since it's still in experimental state till Mojang releases a major update that datapacks moves out of experimental state? Also Dungeons Plus is one of the mods that I've recently downloaded and installed to fill the void that has occured when I started not playing modded minecraft with mods that has incompatible datapacks, even if some mods dont even touch datapacks at all which means I'm either gonna keep playing with this mod or not cause this mod is unfortunately and oddly on the list for mods that has incompatible datapacks, btw for some odd reason that I had Simple Ores installed which adds ores into the game but in the datapacks options the datapack for the mod wasn't saying incompatible but its addon mod called Fusion does have incompatible datapack(s) and all that Fusion does is adds a fusion block that's like a furnace to make different metals like bronze and steel which Fusion doesn't add any ores at all so I really don't understand why Fusion has this problem when it's not a worldgen mod and I'm going to say the same for this mod, this mod doesn't add any worldgen features or content whatsoever but the games datapack system is saying otherwise which is really strange and stupid
